isisd/isis_snmp.c

```diff
@@ -2510,23 +2543,7 @@ static uint8_t *isis_snmp_find_isadj(struct variable *v, oid *name,
 
 	switch (v->magic) {
 	case ISIS_ISADJ_STATE:
-
-		switch (adj->adj_state) {
-		case ISIS_ADJ_UNKNOWN:
-		case ISIS_ADJ_DOWN:
-			val = ISIS_SNMP_ADJ_STATE_DOWN;
```

Comment:
Here there's a small behavior change since `ISIS_ADJ_DOWN` was being mapped to `ISIS_SNMP_ADJ_STATE_DOWN` and now it's being mapped to `ISIS_SNMP_ADJ_STATE_FAILED`. But this is probably what was intended originally. Approving...

Reply:
Yes, I also noticed the difference between two places where this conversion is done. I think making it consistent is the right change.